ci-operator/step-registry/rosa/nodes/wait-ready/rosa-nodes-wait-ready-commands.sh

```diff
@@ -10,24 +10,106 @@ log(){
     echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") " "${*}\033[0m"
 }
 
-# Display details of machines and nodes
+# Display only node details 
 function listNodeDetails() {
+    echo "List node details"
     # Get current machine pools and status of nodes
+    log "$(date) - List infra nodes"
+    echo "oc get nodes -A --no-headers -l node-role.kubernetes.io/infra | cat -n"
+    oc get nodes -A --no-headers -l node-role.kubernetes.io/infra | cat -n
+    log "$(date) - List all worker nodes, excluding infra"
+    echo "oc get nodes --no-headers -l node-role.kubernetes.io/worker,node-role.kubernetes.io/infra!= | cat -n"
+    oc get nodes --no-headers -l node-role.kubernetes.io/worker,node-role.kubernetes.io/infra!= | cat -n
+
+    # Get details of worker nodes not in Ready state
+    log "$(date) - Worker nodes not in Ready state, if any"
+    for node in $(oc get nodes --no-headers -l node-role.kubernetes.io/worker,node-role.kubernetes.io/infra!= --output jsonpath="{.items[?(@.status.conditions[-1].type!='Ready')].metadata.name}"); do
+      oc describe node "$node"
+    done
+    log "$(date) - Finished printing details of all infra and worker nodes."
+}
+
+# Display details of machinesets and nodes for Classic Rosa cluster
+function listMachineAndNodeDetails() {
+    echo "List machine details"
+    # Get current machinesets and machines 
     log "$(date) - List of machinesets"
-    oc get machinesets -A
+    echo "oc get machinesets -A -n openshift-machine-api"
+    oc get machinesets -A -n openshift-machine-api
```

Comment:
since you're specifying -n now, -A is duplicated option

```
    -A, --all-namespaces=false:
	If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even
	if specified with --namespace.
```